quantum/quantum_keycodes.h

```diff
@@ -431,6 +431,18 @@ enum quantum_keycodes {
     // Right shift, close paren
     KC_RSPC,
 
+    // Left control, open paren
+    KC_LCPO,
```

Comment:
Can you move these to the end, just below `SAFE_RANGE`? Otherwise all the keycodes below it will change and this will break VIA.

Reply:
fixed

Reply:
The actual value of the keycodes below here is already uncertain because of the `#ifdef`s around `KC_LEAD` and the faux clicky and MIDI keycodes, so I see little point in splitting these up.

I think some time needs to be spent on allocating this space properly so that VIA doesn't get the rug pulled out from under it like this.